relay/sources/apache/rudder-apache-relay-common.conf

```diff
@@ -17,79 +10,3 @@ DocumentRoot /var/www
 </IfModule>
 </IfModule>
 
-# Expose the server UUID through http
-Alias /uuid /opt/rudder/etc/uuid.hive
-<Directory /opt/rudder/etc>
```

Comment:
isn't this used by the inventory code ?

Reply:
It seems to always use https in 5.0.

Reply:
I also checked usage of relay API, we are using https or ${inventory_protocol} (which is https now) everywhere.